src/main/resources/application.yml

```
@@ -21,8 +21,11 @@ spring:
       ssl:
         enabled: ${REDIS_SSL_ENABLED:false}
 
-  jwt:
-    secret: ${JWT_SECRET_KEY:webee-temporary-secret-key}
-    access-expiration: ${JWT_ACCESS_TOKEN_EXPIRATION:3600000}
-    refresh-expiration: ${JWT_REFRESH_TOKEN_EXPIRATION:86400000}
-    issuer: ${JWT_ISSUER:webee}
+jwt:
+  secret: ${JWT_SECRET_KEY:webee-temporary-secret-key}
+  access-expiration: ${JWT_ACCESS_TOKEN_EXPIRATION:3600000}
+  refresh-expiration: ${JWT_REFRESH_TOKEN_EXPIRATION:86400000}
+  issuer: ${JWT_ISSUER:webee}
+
+kakao:
+  api-key: ${KAKAO_REST_API_KEY}
```

카카오 키가 테스트 환경에서는 없어서 테스트가 실패하네요, 기본값은 공백으로 하면 테스트가 돌아가요!
```suggestion
  api-key: ${KAKAO_REST_API_KEY:}
```